apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -53,6 +57,18 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             ref: 'BillingIntegration',
             isRequired: true,
             many: true,
+            hooks: {
+                validateInput: ({ resolvedData, fieldPath, addFieldValidationError }) => {
+                    if (resolvedData[fieldPath] && !resolvedData[fieldPath].length) {
```

Comment:
if `resolvedData` not an object, then we can get an exception here.
Also if `resolvedData[fieldPath]` is not a string or an array, we also can get exception here.
```
const data = get(resolvedData, fieldPath, '')

if (data ...)
```

Reply:
resolvedData is always object, and since it's many-relation...it's always array (if not - you'll fail during GQL types checking before even entering validateInput function)
<img width="867" alt="image" src="https://user-images.githubusercontent.com/50069872/141248266-7eeea702-6a53-4e7d-8860-aa421eecd5ea.png">